scripts/git/update_tf_submodule.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+set -x
+set -o pipefail
+
+TENSORFLOW_COMMIT="${1:-REMOTE}"
+BRANCH="submodule-update"
+
+git update google
+git checkout -B "${BRANCH?}"
+
+CMD="./scripts/git/update_tf_llvm_submodules.py --llvm_commit=KEEP --update_build_files=true --tensorflow_commit=${TENSORFLOW_COMMIT?}"
```

Comment:
```suggestion
CMD="python3 ./scripts/git/update_tf_llvm_submodules.py --llvm_commit=KEEP --update_build_files=true --tensorflow_commit=${TENSORFLOW_COMMIT?}"
```

Explicit `python3` command has better compatibility on Windows:

```
D:\dev\projects\iree (submodule-update)
λ ./scripts/git/update_tf_llvm_submodules.py --llvm_commit=KEEP --update_build_files=true --tensorflow_commit=${TENSORFLOW_COMMIT?}
'.' is not recognized as an internal or external command,
operable program or batch file.
```

Reply:
What about just dropping the leading `./`. It isn't required, I think

Reply:
No leading `./`, `/` as path separator:
```
λ scripts/git/update_tf_llvm_submodules.py --llvm_commit=KEEP --update_build_files=true
'scripts' is not recognized as an internal or external command,
operable program or batch file.
```

Leading `.\`, `\` as path separator:
```
λ .\scripts\git\update_tf_llvm_submodules.py --llvm_commit=KEEP --update_build_files=true
IREE handy-dandy-LLVM-submodule-updater at your service...
```

Invoke `bash` explicitly on the command, `/` as path separator:
```
λ bash -c "./scripts/git/update_tf_llvm_submodules.py --llvm_commit=KEEP --update_build_files=true"
IREE handy-dandy-LLVM-submodule-updater at your service...
```

Simplest cross-platform support with `python3` or `bash`

Reply:
But the whole script is bash!